css/DesktopVersion.css

```diff
@@ -0,0 +1,365 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;1,100;1,200;1,300;1,400&display=swap");
+
+* {
+  font-family: "Poppins", sans-serif;
+}
+
+@media only screen and (min-width: 768px) {
+  body {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    background-color: #dfe1e6;
+  }
```

Comment:
- Kindly make sure to use the media queries properly according to [this requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md#:~:text=Use%20media%20queries%20in%20your%20CSS%20file%20to%20make%20your%20desktop%20version%20look%20like%20the%20template%20you%20selected%20(Desktop%3A%20%3E%3D768px).). media queries for the Desktop breakpoint in the CSS file, the value for min-width property should be in range of 768px or wider).
-  You app should still be responsive for the mobile version even after implementing the desktop version, that's why I think it would be better to use the media queries in the correct way in order not to cause conflicts between the two versions